components/layout_2020/replaced.rs

```diff
@@ -66,12 +93,21 @@ impl ReplacedContent {
         &self,
         containing_block: &ContainingBlock,
         style: &ComputedValues,
-    ) -> flow_relative::Vec2<Length> {
+    ) -> Vec2<Length> {
         let mode = style.writing_mode();
-        // FIXME(nox): We shouldn't pretend we always have a fully known intrinsic size.
-        let intrinsic_size = self.intrinsic_size.size_to_flow_relative(mode);
-        // FIXME(nox): This can divide by zero.
-        let intrinsic_ratio = intrinsic_size.inline.px() / intrinsic_size.block.px();
+        let intrinsic_size = physical::Vec2 {
+            x: self.intrinsic_width,
+            y: self.intrinsic_height,
+        };
```

Comment:
Why not store the intrinsic size as a `physical::Vec2<Option<_>>`?

Reply:
Could be either. An earlier version of the comment on the field declaration talked about "all three components"